evap/staff/forms.py

```diff
@@ -459,6 +462,12 @@ def clean_weight(self):
             self.add_error("weight", _("At least one evaluation of the course must have a weight greater than 0."))
         return weight
 
+    def clean_main_language(self):
+        main_language = self.cleaned_data.get("main_language")
+        if self.requires_decided_main_language and main_language == Evaluation.UNDECIDED_MAIN_LANGUAGE:
+            self.add_error("main_language", _("You have to set a main language to approve this evaluation."))
```

Comment:
This form does not know about approving evaluations, so we should make this message less specific. If we _really_ want to mention this approving here, we should change the `requires_decided_main_language` to something like `decided_main_language_reason: StrOrPromise | None` and pass in this string from the outside